versioned_docs/version-2.x/magento2/b2b.mdx

```diff
@@ -106,3 +107,59 @@ entries (company management, credit history…).
 
 To enable _Payment on account_ payment method, please refer to
 [the Payment on account guide](/docs/2.x/advanced/payments/payment-on-account).
+
+## Negotiable quotes
+
+<SinceVersion tag="2.23" />
+
+In order to use Front-Commerce's Negotiable quotes module, you will need to
+enable the feature in your Magento back office.
+
+Navigate to `Stores > Configuration > General > B2B Features`, and set both
+`Enable Company` and `Enable B2B quote` to `Yes`.
+
+Quotes also have to be enabled _per-company_. To do so, navigate to a company's
+settings (`Customers > Companies`, then open a company), and under
+`Advanced settings` switch `Allow Quotes` on.
+
+:::info
+
+Negotiable quotes are only availble to users that have a Company with Quotes
+enabled. More in
+[Adobe documentation](https://experienceleague.adobe.com/docs/commerce-admin/b2b/quotes/configure-quotes.html).
+
+:::
+
+That's it, you should now be able to use Adobe Commerce's Negotiable quotes with
+Front-Commerce !
+
+### Known issues
+
+This module has been fully implemented with Adobe Commerce's GraphQL API
+(version 2.4.5). Unfortunately, this means it is also limited to what the API
+exposes, thus some of the features available in Adobe Commerce's native frontend
+could not be tackled yet:
+
+> Negotiable quotes miminum price is not enforced nor exposed in the GraphQL API
```

Comment:
**nitpick:** having a title would allow to have an anchor to create links directly to this section (could be useful for support)

```suggestion
#### Negotiable quotes miminum price is not enforced nor exposed in the GraphQL API
```